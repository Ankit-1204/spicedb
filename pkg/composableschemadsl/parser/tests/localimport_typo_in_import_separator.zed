from .path.to.user impoort user, persona

definition resource {
    relation user: user
    relation persona: persona
    permission view = user + persona
}
