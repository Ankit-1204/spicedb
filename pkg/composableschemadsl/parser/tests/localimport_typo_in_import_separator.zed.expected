NodeTypeFile
  end-rune = 17
  input-source = local imports with typo in import separator test
  start-rune = 0
  child-node =>
    NodeTypeImport
      end-rune = 17
      input-source = local imports with typo in import separator test
      start-rune = 0
      child-node =>
        NodeTypeError
          end-rune = 17
          error-message = Expected keyword import, found token TokenTypeIdentifier
          error-source = impoort
          input-source = local imports with typo in import separator test
          start-rune = 19
      path-segment =>
        NodeTypeIdentifier
          end-rune = 9
          identifier-value = path
          input-source = local imports with typo in import separator test
          start-rune = 6
        NodeTypeIdentifier
          end-rune = 12
          identifier-value = to
          input-source = local imports with typo in import separator test
          start-rune = 11
        NodeTypeIdentifier
          end-rune = 17
          identifier-value = user
          input-source = local imports with typo in import separator test
          start-rune = 14
    NodeTypeError
      end-rune = 17
      error-message = Unexpected token at root level: TokenTypeIdentifier
      error-source = impoort
      input-source = local imports with typo in import separator test
      start-rune = 19